polyfactory/factories/pydantic_factory.py

```diff
@@ -101,6 +102,10 @@
 _IS_PYDANTIC_V1 = VERSION.startswith("1")
 
 
+class BuildContext(BaseBuildContext):
```

Comment:
Could we rename this to `PydanticBuildContext` in order to keep it consistent with `PydanitcConstraints`?

Reply:
Sure, thanks!